packages/salesforcedx-sobjects-faux-generator/src/messages/i18n.ts

```diff
@@ -26,7 +26,7 @@ export const messages = {
   unsupported_sobject_category:
     'SObject category cannot be used to generate metadata %s',
   no_generate_if_not_in_project:
-    'Unable to process sObjects when not in an SFDX project %s',
+    'Unable to process sObjects when not in an SF project %s',
```

Comment:
**Can you change the term "SF Project" to "Salesforce Project"?**

I think of projects as either "SFDX Projects" or "Salesforce Projects".  I'm going with this reasoning because we say "SFDX CLI" for SFDX v7 and "Salesforce CLI" for SF v2, and also the commands in the command palette still say SFDX.  In my mind, "SF" is the acronym for San Francisco rather than Salesforce so "SF Project" makes me think "San Francisco Project" - in fact, this acronym confusion is one of the reasons why we decided to keep the SFDX name for the command palette. 😂